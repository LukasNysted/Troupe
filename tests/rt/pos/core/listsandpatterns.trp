let fun f (a,b) [] = 0 
      | f _ [] = 1 
 in (f ([],[]) []) + (f ([],[]) [])
 end 