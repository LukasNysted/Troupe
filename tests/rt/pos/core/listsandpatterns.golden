2021-06-12T12:08:27.276Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: 0@{}%{}
